frontend/README.md

```diff
@@ -26,6 +26,21 @@ You can choose your own preferred webserver like for example `nginx`, `serve` or
    npm run dev
    ```
 
+## Setting up the environment variables
+
+The project requires a couple of environment variables to run, if you want to develop on this codebase.
+Setting values for these variables can be done with a method to your own liking.
```

Comment:
Ok, but maybe explain how we do it so they can use that, and are you sure this can be done with a method to their liking? 'Cause our code always works with a .env file or docker env variables right?

Reply:
This line in general is not necessary